application/tests/codeception/common/templates/fixtures/user.php

```diff
@@ -1,8 +1,8 @@
 <?php
 /**
- * @var $faker \Faker\Generator
- * @var $index integer
- */
+     * @var $faker \Faker\Generator
```

Comment:
that is why I finally accepted them. Since these are files auto-generated by codeception and scrutinizer kept wanting to indent them, and even ignoring the suggestion kept it on the list so I just accepted to get them out of the way since they dont actually break anything.